js/rpg_scenes/Scene_Boot.js

```diff
@@ -23,20 +23,20 @@ Scene_Boot.prototype.create = function() {
 };
 
 Scene_Boot.prototype.loadSystemWindowImage = function() {
-    ImageManager.loadSystem('Window');
+    ImageManager.reserveSystem('Window');
 };
 
 Scene_Boot.loadSystemImages = function() {
-    ImageManager.loadSystem('IconSet');
-    ImageManager.loadSystem('Balloon');
-    ImageManager.loadSystem('Shadow1');
-    ImageManager.loadSystem('Shadow2');
-    ImageManager.loadSystem('Damage');
-    ImageManager.loadSystem('States');
-    ImageManager.loadSystem('Weapons1');
-    ImageManager.loadSystem('Weapons2');
-    ImageManager.loadSystem('Weapons3');
-    ImageManager.loadSystem('ButtonSet');
+    ImageManager.reserveSystem('IconSet');
+    ImageManager.reserveSystem('Balloon');
+    ImageManager.reserveSystem('Shadow1');
+    ImageManager.reserveSystem('Shadow2');
+    ImageManager.reserveSystem('Damage');
+    ImageManager.reserveSystem('States');
+    ImageManager.reserveSystem('Weapons1');
+    ImageManager.reserveSystem('Weapons2');
+    ImageManager.reserveSystem('Weapons3');
+    ImageManager.reserveSystem('ButtonSet');
```

Comment:
Hmm I doubt that it will work, because these reservation will release on `Scene_Boot` terminated.